src/components/radio-button/radio-button.mdx

```diff
@@ -52,6 +52,8 @@ import {
 
 A legend can be set for the group, and each radio button can have a label.
 
+**Note**: The `labelHelp` prop is not supported if the `validationRedesignOptIn` flag on the `CarbonProvider` is true.
+
 <Canvas of={RadioButtonStories.WithLegendAndLabels} />
```

Comment:
**suggestion(non-blocking)**: what do you about updating the original story to use the new validation designs by default? This could reduce confusion for consumers who might overlook the statement above, ensuring they clearly see what the latest behaviour in Carbon should be.